tests/nnfw_api/src/GenModelTests/nontrainable_op_trains/MaxPool2D.test.cc

```diff
@@ -168,5 +168,95 @@ TEST_F(GenModelTrain, NonTrainableOps_Conv2D_MaxPool2D_Depth2_Filter2)
   }
 }
 
-// TODO : Add test case to check stride supporting
-// TODO : Add more negative test cases
+TEST_F(GenModelTrain, NonTrainableOps_Conv2D_MaxPool2D_Stride2Filter2)
+{
```

Comment:
```
Epoch 1/5
1/1 [==============================] - 0s 239us/step - loss: 25.5000
Epoch 2/5
1/1 [==============================] - 0s 158us/step - loss: 19.2126
Epoch 3/5
1/1 [==============================] - 0s 155us/step - loss: 12.9202
Epoch 4/5
1/1 [==============================] - 0s 134us/step - loss: 9.0784
Epoch 5/5
1/1 [==============================] - 0s 147us/step - loss: 6.6907
```

Reply:
(note) I made a PR, but this code is originally written by @ragmani. 
Most of code is copy-past from https://github.com/ragmani/ONE/blob/3920c734227f8f5aa18cfd994293abb41363f447/tests/nnfw_api/src/GenModelTests/nontrainable_op_trains/MaxPool2D.test.cc